package.json

```
@@ -1,6 +1,9 @@
 {
   "author": "Doka.Team",
   "license": "MIT + CC BY-NC-SA 4.0",
+  "engines": {
+    "node": ">=14.0.0 < 15.0.0"
+  },
```

```suggestion
  "engines": {
    "node": "14.x",
    "npm": "6.x"
  },
```

А может так? Дело в том, что лок-файл v2 приезжает с npm@7, которую можно поставить и с Node.js 12, наверное. Тут сразу явно: работаем с такой и такой версиями.

---

Согласен. Смотрел не в тот релизный репозиторий `npm` и не нашел там семерку